packages/core/src/components/Chip/index.tsx

```diff
@@ -7,8 +7,12 @@ import ProfilePhoto from '../ProfilePhoto';
 import ButtonOrLink, { ButtonOrLinkTypes } from '../private/ButtonOrLink';
 
 export type Props = {
+  /** Renders with a primary background and white text */
```

Comment:
Nit, but could you add a period to the end of these sentences.

Reply:
There were 2 of them :P

Reply:
😭